shared/pingpong/accounts_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package pingpong
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"github.com/algorand/go-algorand/crypto"
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/stretchr/testify/assert"
+)
+
+func makeKeyFromSeed(i uint64) *crypto.SignatureSecrets {
```

Comment:
Lift `MakeKeyFromSeed(uint64)` to the other place this code is implemented and use the function in that place and here.

Reply:
yes I was weighing whether to make the other implementation in deployedNetwork.go an exported function or not, and cut corners duplicating it for the test ... will fix